src/Sarif.Viewer.VisualStudio.Core/Models/SarifErrorListItem.cs

```diff
@@ -951,5 +974,32 @@ private static string FormatNotficationText(Notification notification)
 
             return message;
         }
+
+        /// <summary>
+        /// Creates the content that goes into the ToolTipContent that gets rendered on hover.
+        /// Will attempt to render markdown first, however if it fails it will fall back to plaintext.
+        /// </summary>
+        /// <returns>ToolTipContent to render.</returns>
+        private List<(string strContent, TextRenderType renderType)> CreateContent()
+        {
+            if (_content == null)
+            {
+                _content = new List<(string strContent, TextRenderType renderType)>();
+                if (this.SarifResult != null && this.SarifResult.Message != null && !string.IsNullOrWhiteSpace(this.SarifResult.Message.Markdown))
```

Comment:
>this.SarifResult != null && this.SarifResult.Message != null && !string.IsNullOrWhiteSpace(this.SarifResult.Message.Markdown) [](http://example.com/codeflow?start=20&length=125)

I think you can smoosh this with the null conditional, e.g.
if(!string.IsNullOrWhiteSpace(this.SarifResult?.Message?.Markdown))